tests/scripts/get_token_metadata.cdc

```diff
@@ -0,0 +1,16 @@
+import ExampleToken from "ExampleToken"
+import FungibleTokenMetadataViews from "FungibleTokenMetadataViews"
+import MetadataViews from "MetadataViews"
+
```

Comment:
Can you please add a short description at the top of each transaction in a comment? Thanks!

Reply:
Added in https://github.com/onflow/flow-ft/pull/137/commits/d7a27dc816bf01cad2082ca4aeaec2b781e9d969